summary/table.go

```diff
@@ -100,42 +100,42 @@ func DisplaySummaryOutput(resp *opb.Response, revDNSLookup bool, requestType str
 	}
 
 	if strings.Contains(requestType, "network") {
-		if len(resp.InNwData) > 0 {
+		if len(resp.IngressConnection) > 0 {
 			fmt.Printf("\nIngress connections\n")
 			// Display server conn data
 			inNwRowData := [][]string{}
-			for _, inNwData := range resp.InNwData {
+			for _, IngressConnection := range resp.IngressConnection {
 				inNwStrSlice := []string{}
```

Comment:
Hey @yasin-cs-ko-ak , here if you notice so we have two variables inNwData and InNwData, where InNwData is part of observability protobuf. You replaced both of the variables with a single "IngressConnection" variable. Please take a look at it.

Reply:
@Ankurk99 Thank you for the review. Changes are done.